src/src/components/form/CustomSelectInput.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+    <div :class="classname">
+        <label v-if="prefix(field)" v-html="prefix(field)"></label>
+        <select
+            :name="select.propName"
+            :id="select.id"
+            :disabled="select.disabled"
+            :value="selectedValueState"
+            @change="$emit('selectValueChanged', selectedValueState)"
+        >
+            <option :v-if="select.showDefault" value disabled selected>
+                {{ select.defaultLabel }}
+            </option>
+            <option
+                v-for="(opt, index) in select.propOptions"
+                :key="index"
+                :value="opt.propVal"
+                >
+                {{ opt.propName }}
+            <span :v-if="opt.countForOption">({{ opt.countForOption }}) </span>
+            </option>
+        </select>
+        <label v-if="suffix(field)" v-html="suffix(field)"></label>
+    </div>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, PropType, ref } from 'vue';
+import SelectModel from '../../models/FormModels';
+
+export default defineComponent({
+    name: 'FormSelect',
+    props: {
+      select: {
+        type: Object as PropType<SelectModel>,
+        required: true,
+      },
+    },
+    emits: ['selectValueChanged'],
+    setup(props) {
+      const selectedValueState = ref<number>();
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 6.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
    const selectedValueState = ref<number>();
```